tests/drivers/i2c/i2c_sensor/src/main.c

```diff
@@ -0,0 +1,293 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/drivers/i2c.h>
+#include <zephyr/kernel.h>
+#include <zephyr/ztest.h>
+#include "sensor.h"
+
+#if DT_NODE_HAS_STATUS(DT_ALIAS(sensor_bme688), okay)
+#define I2C_TEST_NODE DT_ALIAS(sensor_bme688)
+#define DEVICE_ADDRESS (uint8_t)DT_REG_ADDR(DT_CHILD(DT_ALIAS(sensor_bme688), bme688_76))
+#elif DT_NODE_HAS_STATUS(DT_NODELABEL(i2c1), okay)
+#define I2C_TEST_NODE DT_NODELABEL(i2c1)
+#define DEVICE_ADDRESS (uint8_t)DT_REG_ADDR(DT_NODELABEL(i2c1, bme688_76))
+#else
+#error "Unable to get I2C device"
+#endif
```

Comment:
With the change to `testcase.yaml`, this should be changed to use something like `DT_COMPAT_GET_ANY_STATUS_OKAY()` instead of relying on aliases.

Reply:
Changed to:

#define SENSOR_NODE DT_COMPAT_GET_ANY_STATUS_OKAY(bosch_bme680)
#define I2C_TEST_NODE DT_PARENT(SENSOR_NODE)
#define DEVICE_ADDRESS (uint8_t)DT_REG_ADDR(SENSOR_NODE)